docs/api/puppeteer.awaitablepredicate.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_label: AwaitablePredicate
```

Comment:
This is a nice change, it would be good to extract it.This is a nice change, it would be good to extract it.